chaco/color_mapper.py

```diff
@@ -405,7 +404,7 @@ def _map(self, X):
 
         """
 
-        if type(X) in [IntType, FloatType]:
+        if type(X) in [int, float]:
```

Comment:
Orthogonal to the PR (so please don't change it here), but it bugs me that this isn't `if instance(X, (int, float, long))`. Is it worth opening an issue to remind us to fix these explicit type checks? Or should we just wait for it to cause problems in something real?